lib/cdo/contact_rollups.rb

```diff
@@ -963,4 +949,139 @@ def self.grab_next(s)
   rescue StandardError => error
     log "Error iterating over stream #{s} - #{error}"
   end
+
+  def self.initialize_connections_to_database_clone
```

Comment:
These functions seems to be exactly the same as the ones in https://github.com/code-dot-org/code-dot-org/pull/28767/?